mobile-ui/build.gradle

```diff
@@ -1,3 +1,7 @@
+plugins {
```

Comment:
What is purpose of this plugin? Is it related to `Migrate to androidX` MR?

Reply:
It's used to check if new versions of libraries are available. No needed for androidx. You can skip it.